calico/network-policy/policy-tiers/rbac-tiered-policies.mdx

```diff
@@ -0,0 +1,507 @@
+---
+description: Configure RBAC to control access to policies and tiers.
+---
+
+# Configure RBAC for tiered policies
```

Comment:
I suppose this is just a plain copy from EE, isn't it?

How does the RBAC play with ANP tier? Is there some RBAC created by default or do kubernetes handle it on their own? Idk how it works, just double checking here!

Reply:
FYI @mazdakn

Reply:
Yes, this is just a copy from EE. 
ANP design states the API is for cluster administrators. This is not something we enforce though. We provide Tier's RBAC for users to configure the access, so in the end it's up to users to setup the access properly.